gtest_parallel.py

```diff
@@ -678,11 +689,19 @@ def main():
 
   parser = default_options_parser()
   (options, binaries) = parser.parse_args()
+
+  if (options.output_dir is not None and
+      not os.path.isdir(options.output_dir)):
+    parser.error('--output_dir value must be an existing directory, '
+                 'current value is "%s"' % options.output_dir)
+
   # Append gtest-parallel-logs to log output, this is to avoid deleting user
   # data if an user passes a directory where files are already present. If a
   # user specifies --output_dir=Docs/, we'll create Docs/gtest-parallel-logs
   # and clean that directory out on startup, instead of nuking Docs/.
-  options.output_dir = os.path.join(options.output_dir, 'gtest-parallel-logs')
+  if options.output_dir:
+    options.output_dir = os.path.join(options.output_dir,
```

Comment:
So this is what you're talking about.
Yeah, wouldn't normally expect `options` to change.

Feel free to disregard my other comments

Reply:
Yeah sorry this is a little hard to read, we update the existing variable to prevent the rest of the code from accidentally reading the old value and mixing up `output_dir` with `output_dir/gtest-parallel-logs`.